lib/perl/Genome/VariantReporting/Suite/BamReadcount/MaxVafObservedInterpreter.pm

```diff
@@ -53,14 +53,16 @@ sub _interpret_entry {
     my @vaf_hash_names        = (\%normal_vafs,        \%tumor_vafs);
     my $it                    = each_array(@sample_name_accessors, @vaf_hash_names);
     while ( my ($sample_name_accessor, $vaf_hash_ref) = $it->() ) {
-        for my $sample_name ($self->$sample_name_accessor) {
+        if (defined($self->$sample_name_accessor)) {
```

Comment:
if `$sample_name_accessor` is returning a list, then it will always be `defined` in a scalar context.  (With no values it would return `0`.)